.chloggen/35081-fix-iops-search.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'bug_fix'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: splunkenterprise
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: fix for a flaky search present in the splunkenterprise receiver related to iops metrics.
```

```suggestion
note: Fix a flaky search related to iops metrics.
```
The changelog note will be prepended with the component name, so we don't need to include it here as well. 👍